### PR TITLE
chore(core): enable native hasher for ci

### PR DESCRIPTION
**Pull request by Cammisuli**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: nx.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "affected": {
     "defaultBase": "master"
   },
+  "//": "",
```
Comment (AgentEnder) [inline nx.json:6]: What's up with this?

Comment (Cammisuli) [reply nx.json:6]: Wanted to invalidate the whole cache. It's not going to stay there

**Review by bcabanes — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -218,6 +222,7 @@ jobs:
     environment:
       NX_E2E_CI_CACHE_KEY: e2e-circleci-macos
       NX_DAEMON: 'true'
+      NX_PERF_LOGGING: 'true'
```
Comment (FrozenPandaz) [inline .circleci/config.yml:225]: Shouldn't this have the native hasher as well?

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (Cammisuli) [reply .circleci/config.yml:225]: Yes, you're right.

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**